tag/nitag.yml

```diff
@@ -0,0 +1,1816 @@
+  - application/json
+securityDefinitions:
+  apiKeyAuth:
+    type: apiKey
+    name: x-ni-api-key
+    in: header
+  basicAuth:
+    type: basic
+security:
+  - apiKeyAuth: []
+  - basicAuth: []
+
+responses:
+  Unauthorized:
+    description: Not authorized
+    headers:
+      WWW_Authenticate:
+        description: Information for how to authenticate (sent only from SystemLink Server).
+        type: string
+  Error:
+    description: Error (sent only from SystemLink Server)
+    schema:
+      description: Error response
+      title: ErrorResponse
+      type: object
+      properties:
+        error:
+          $ref: '#/definitions/Error'
+  GetTagsWithValuesResponse:
+    description: List of tags, including their current and aggregate values
+    schema:
+      properties:
+        totalCount:
+          description: Total number of tags which match a given query
+          type: integer
+        tagsWithValues:
+          description: List of tags with their current and aggregate values
+          type: array
+          items:
+            $ref: '#/definitions/TagWithValue'
+  GetTagsResponse:
+    description: List of tags, excluding current and aggregate values
+    schema:
+      properties:
+        totalCount:
+          type: integer
+          description: Total number of tags which match a given query
+        tags:
+          description: List of tags
+          type: array
+          items:
+            $ref: '#/definitions/Tag'
+  GetSubscriptionUpdatesResponse:
+    description: List of subscription updates
+    schema:
+      properties:
+        subscriptionUpdates:
+          description: Subscription updates
+          type: array
+          items:
+            $ref: '#/definitions/SubscriptionUpdate'
+        invalidSubscriptionIds:
+          description: List of invalid subscription IDs
+          type: array
+          items:
+            type: string
+
+paths:
+  /v2/subscriptions:
+    post:
+      tags:
+        - subscriptions
+      summary: Create a subscription
+      description: Creates a subscription for a list of tags to get notifications when the tags change.
+      operationId: CreateSubscription
+      parameters:
+        - in: body
+          name: subscription
+          description: Subscription to create
+          schema:
+            $ref: '#/definitions/Subscription'
+          required: false
+      responses:
+        200:
+          description: Success
+          schema:
+            description: Response for creating a subscription
+            type: object
+            title: Create Subscription Response
+            properties:
+              subscriptionId:
+                type: string
+              nonexistentTags:
+                description: List of tags which did not exist
+                type: array
+                items:
+                  type: string
+              alias:
+                type: string
+                description: Name of the subscription
+        400:
```

Comment:
Do we need to document the 400 explicitly? SL Server would include an Error response in a 400, which is covered by the default case.

Reply:
Probably not, falling back to the default error response is probably good enough.